packages/paymaster/src/types/Types.ts

```diff
@@ -4,3 +4,9 @@ export type PaymasterAndDataResponse = {
     paymasterAndData: string
   }
 }
+
+export type PaymasterConfig = {
+  paymasterServiceUrl: string,
+  strictSponsorshipMode?: boolean,
+  dappAPIKey: string
```

Comment:
Don't use the name dappAPIKey. Just use the name `apiKey`

Reply:
this will soon be obsolete we will just use paymasterUrl